functional_test.go

```diff
@@ -341,8 +341,8 @@ func TestTokenBucketNegativeHits(t *testing.T) {
 			resp, err := client.GetRateLimits(context.Background(), &guber.GetRateLimitsReq{
 				Requests: []*guber.RateLimitReq{
 					{
-						Name:      "test_token_bucket",
-						UniqueKey: "account:1234",
+						Name:      "test_token_bucket_negative",
+						UniqueKey: "account:12345",
```

Comment:
Now that I see this was a problem, I would recommend not using arbitrary key names that could be reused by other tests.  Instead of `12345` create a const value with a unique descriptive key name.

Reply:
nvm, you already did with a unique `Name`.  No problem here.